### PR TITLE
Reference seed projects for Ionic 1 and ASP.NET Core quickstarts

### PR DESCRIPTION
**Pull request by jerriep**
Reference seed projects for the following quickstarts
- Ionic 1
- ASP.NET Core MVC
- ASP.NET Core Web API

I also removed an old file which should not be part of the ASP.NET Core Web API quickstart anymore

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by mpaktiti** [comment]
Hi @jerriep there are some conflicts, can you look into it? Thanks.

**Comment by jerriep** [comment]
@mpaktiti Strange.. Seems there were some subsequent commits which have been committed which now causes the conflicts

I will look into it

**Comment by jerriep** [comment]
@mpaktiti I did a rebase

File: articles/native-platforms/ionic/00-intro.md [hunk 1]
```diff
@@ -3,8 +3,28 @@ title: Introduction
...
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples/tree/master/00-Starter-Seed',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '00-Starter-Seed',
+  pkgFilePath: '00-Starter-Seed/www/app/auth0.variables.js',
+  pkgType: 'replace'
+}) %>
+
 Auth0 provides and manages an [AngularJS SDK](https://github.com/auth0/auth0-angular) which can be used with Ionic applications. This SDK wraps Auth0's [Lock Widget](https://github.com/auth0/lock) as well as the [Auth0.js library](https://github.com/auth0/auth0.js), so it makes it simple to integrate these in your Ionic applications.
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/00-intro.md:18]: @jerriep Can you remove please one of the includes? https://auth0-docs-content-pr-1963.herokuapp.com/docs/quickstart/native/ionic

File: articles/server-apis/aspnet-core-webapi/00-intro.md [hunk 2]
```diff
@@ -3,10 +3,30 @@ title: Introduction
...
+  githubUrl: 'https://github.com/auth0-samples/auth0-aspnetcore-webapi-sample/tree/master/00-Starter-Seed',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-aspnetcore-webapi-sample',
+  pkgBranch: 'master',
+  pkgPath: '00-Starter-Seed',
+  pkgFilePath: null,
+  pkgType: 'server'
+}) %>
+
 If you would like to follow along with this Quickstart you can download the [seed project](https://github.com/auth0-samples/auth0-aspnetcore-webapi-sample/tree/master/00-Starter-Seed). The seed project is just a basic ASP.NET Web API with a simple controller and some of the NuGet packages which will be needed included. It also contains an `appSettings.json` file where you can configure the various Auth0-related settings for your application.
```
Comment (mpaktiti) [inline articles/server-apis/aspnet-core-webapi/00-intro.md:20]: @jerriep Can you please remove one of the two includes? https://auth0-docs-content-pr-1963.herokuapp.com/docs/quickstart/backend/aspnet-core-webapi

**Comment by mpaktiti** [comment]
Hi @jerriep can you downsize this one a bit? `media/articles/tokens/dashboard-revoke-refresh-token.png` It's 2424 × 1642. Check also that it has been through tinypng.com . Thanks

**Comment by jerriep** [comment]
@mpaktiti Sorry, I do not know what went on with this PR. It seems there were multiple PRs accepted which touched the same code and things went haywire

The image you referenced was not part of my docs, but I resized it

**Pull request merged**